nebulacli.py

```diff
@@ -148,7 +148,14 @@ def load_config_from_path(file):
 
 
 def get_instance_id():
-    r = requests.get('http://169.254.169.254/latest/meta-data/instance-id')
+    imdsv2_token_url = 'http://169.254.169.254/latest/api/token'
+    token_request_headers = {'X-aws-ec2-metadata-token-ttl-seconds': '21600'}
+    t = requests.put(imdsv2_token_url, headers=token_request_headers)
+    t.raise_for_status()
+    imdsv2_token = t.text
+
+    metadata_request_headers = {'X-aws-ec2-metadata-token': imdsv2_token}
+    r = requests.get('http://169.254.169.254/latest/meta-data/instance-id', headers=metadata_request_headers)
     r.raise_for_status()
     return r.text
```

Comment:
This switches from v1 to v2, but that means it'll break for other users (there are some, I swear!) who haven't switched over yet.

WHat I would do is attempt to call v1 first, and then check the status. If the status is successful then it can return the instance id immediately, otherwise it can attempt v2. This way if people have v1 available it'll work, but if they're enforcing v2 that will work.

Reply:
Adding IMDSv1 attempt first